cypress-tests/cypress/fixtures/create-confirm-body.json

```diff
@@ -1,10 +1,10 @@
 {
-  "amount": 6540,
+  "amount": 6500,
```

Comment:
Can we have amount also from connector configs?

Reply:
this is being added in a refactor, would go in  another pr

Reply:
sure!